footer.php

```diff
@@ -1,24 +1,5 @@
-<?php
-
-?>
-
-<!DOCTYPE html>
-<html>
-<head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
-<title>FUCK THE JETS</title>
-
-</head>
-	<style>
-		.contribute {
-			position: relative;
-			font-family: "Arial", Gadget, sans-serif;
-		}
-	</style>
-	<body>
-		
-		<div class="contribute">
-                        <h3 style="position: fixed; bottom: 0; width:100%; text-align: center"><span style="font-size:12px;">Contribute to this website <a href="https://github.com/FuckTheJets/fuckthejets">here</a></span><h3>
-		</div>
-		
-	</body>
-</html>
+<div class="contribute">
+    <a href="https://github.com/FuckTheJets/fuckthejets">
+        <h3>Contribute to this website<h3>
+    </a>
+</div>
```

Comment:
You can think of PHP includes as copying the content of the file you're including and pasting it where you're calling the include - no need for `doctype`, `html` or `body` tags